libs/platform/ebpf_platform.h

```diff
@@ -458,6 +471,24 @@ extern "C"
     ebpf_result_t
     ebpf_hash_table_next_key(ebpf_hash_table_t* hash_table, const uint8_t* previous_key, uint8_t* next_key);
 
+    /**
+     * @brief Returns the next (key, value) pair in the hash table.
+     *
+     * @param[in] hash_table Hash-table to query.
+     * @param[in] previous_key Previous key or NULL to restart.
+     * @param[out] next_key Next key if it exists.
+     * @param[out] next_value Next value if it exists.
```

Comment:
```suggestion
     * @param[out] next_value If non-NULL, returns the next value if it exists.
```
Since line 490 says _In_opt_ then null is legal and the description should say so

Reply:
Done.